ui/app/styles/components/codemirror.scss

```diff
@@ -168,31 +168,12 @@ $gutter-grey: #2a2f36;
 }
 
 .readonly-codemirror {
-  .CodeMirror-cursors {
```

Comment:
this was not working. We have it working now under a new class that only shows if the editor is readOnly.